include/Player.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "Prelude.h"
+#include "Shuffle.h"
+
+#define MAX_NUMBER_OF_BLINDFOLDS 6
+#define NUMBER_OF_BLINDFOLDS 20
+
+enum SpecialBlindfold
+{
+	HumanEyes = 0,
+	FoxEyes = 1,
+	ScorpionEyes = 2
+};
+
+struct Player
+{
+	Bool is_pushing;
+	Bool all_push_plates_checked;
```

Comment:
Maybe we could move this to another component e.g. Game or Level... because if the player moves all the boxes to the correct platforms the level is over or some action occurs. It makes more sense to me that it should be in the component related to the game or the level itself.

Reply:
It makes perfect sense :)